content/docs/accessibility.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: accessibility
-title: Accessibility
+title: Հասանելիություն
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## Ինչու՞ հասանելիություն {#why-accessibility}
 
-Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.
+Վեբ հասանելիությունը (հղվում է նաև [**a11y**](https://en.wiktionary.org/wiki/a11y)-ին) այնպիսի վեբ կայքի կառուցվածքն ու պատրաստումն է, որը կարող է օգտագործել յուրաքանչյուրը։ Հասանելիության ապահովումը անհրաժեշտ է, որպեսզի օգնական տեխնոլոգիաները հնարավորություն ունենան վեբ էջերը ինտերպրետացնելու։
```

Comment:
the design - նախագծումը (գրած ա կառուցվածքը)
create - ստեղծել (պատրաստելը make-ն ա)
websites - կայքեր (գրած ա կայք)

...այնպիսի վեբ կայքերի նախագծումն ու ստեղծումն է, որոնք կարող են օգտագործվել յուրաքանչյուրի կողմից։